### DISCUSSION TITLE
Sounds in Korge 5

### DISCUSSION THREAD

**Discussion by Kutyrev** [korlibs/korge#2043] [Q&A]
Good day, dear Korge team!

I have moved my project from Korge 4.0.9 to 5.1.0 and noticed that the sounds stopped playing.

Works in 4.0.9, does not work in 5.1.0:
 ```
class MyScene : Scene() {

    private var clickSound : Sound? = null

    override suspend fun SContainer.sceneInit() {
        clickSound = resourcesVfs["click.mp3"].readSound()
    }

    override suspend fun SContainer.sceneMain() {
        uiVerticalStack {
            uiButton("New game").onClick {
                clickSound?.play()
            }
        }.centerOnStage()
    }
}
 ```

But if I change readSound() to readMusic() — everything fine again.
Is this a bug or do I need to somehow change my solution for the sounds to work correctly?

**Comment by soywiz** [comment]
Looks like a bug. I hav ecreated this issue: https://github.com/korlibs/korge/issues/2104

**Comment by FSaurenbach** [comment]
Fixed in 6.0.0-beta1
